test/index.test.ts

```diff
@@ -114,10 +115,31 @@ describe('index', () => {
 			expect(amqp.connect).to.have.been.calledWithExactly('amqp://localhost');
 			expect(connection.createChannel).to.have.been.calledTwice;
 			expect(channel.prefetch).to.not.have.been.called;
+			expect(channel.close).to.have.been.calledTwice;
 			expect(connection.close).to.have.been.calledOnce;
 
 		});
 
+		it('should ignore closing the connection and channels if connect has not been called', async () => {
+
+			// Given
+			transport = new Transport({
+				url: 'amqp://localhost'
+			});
+
+			// When
+			await transport.close();
+
+			// Then
+			expect(optionsValidator.validate).to.have.been.calledOnce;
+			expect(amqp.connect).to.not.have.been.called;
+			expect(connection.createChannel).to.not.have.been.called;
+			expect(channel.prefetch).to.not.have.been.called;
+			expect(channel.close).to.have.not.been.calledOnce;
```

Comment:
This will pass if the stub is called 0, or 2 or more times. Should be not.called.
```suggestion
			expect(channel.close).to.have.not.been.called;
```

Reply:
Modified